cmd/kubeadm/app/apis/kubeadm/v1beta1/types.go

```diff
@@ -106,12 +106,23 @@ type ClusterConfiguration struct {
 	// CertificatesDir specifies where to store or look for all required certificates.
 	CertificatesDir string `json:"certificatesDir"`
 
-	// ImageRepository what container registry to pull control plane images from
+	// ImageRepository what container registry to pull control plane images from.
```

Comment:
This should probably now state `... to pull images from.` (remove the `control plane` part of it). Since this could be used for kube-proxy and pause images, which run on worker nodes too.